File: Trabalho de programação/MediaAcimaDaMedia.cs
using System;
using System.Collections.Generic;

class Program {
    static void Main() {
        Console.Write("Quantos números você deseja inserir? ");
        int n = int.Parse(Console.ReadLine());

        List<int> numeros = new List<int>();
        int soma = 0;

        for (int i = 0; i < n; i++) {
            Console.Write("Insira o número {0}: ", i + 1);
            int numero = int.Parse(Console.ReadLine());
            numeros.Add(numero);
            soma += numero;
        }

        float media = (float)soma / n;
        Console.WriteLine("Média: {0:F2}", media);
        Console.WriteLine("Números acima da média:");

        foreach (int numero in numeros) {
            if (numero > media) {
                Console.Write(numero + " ");
            }
        }
    }
}

File: Trabalho de programação/media_acima_da_media.py
def main():
    n = int(input("Quantos números você deseja inserir? "))
    
    numeros = []
    soma = 0

    for i in range(n):
        numero = int(input(f"Insira o número {i + 1}: "))
        numeros.append(numero)
        soma += numero

    media = soma / n
    print(f"Média: {media:.2f}")
    print("Números acima da média:")

    for numero in numeros:
        if numero > media:
            print(numero, end=" ")

if __name__ == "__main__":
    main()
